docs/how-to-guides/block-tutorial/nested-blocks-inner-blocks.md

```diff
@@ -155,13 +155,15 @@ add_action( 'init', function() {
 } );
 ```
 
-## Child InnerBlocks: Parent and Ancestors
+## Using Hierarchical Context in InnerBlocks
```

Comment:
I personally (maybe because I'm not a native English speaker) fund Hierarchical Context to be much more difficult to understand than Parent and Ancestor. I would love to better understand what the concerns / struggles are with that verbiage that are behind this change :)

Reply:
> find Hierarchical Context to be much more difficult to understand than Parent and Ancestor.

I agree. `Hierarchical Context” is clumsy. I’m mostly struggling with the previous “Child InnerBlocks”. That seem clumsy and the “Child” seems overqualifying. Maybe we just drop the “Child” from the heading and go with “InnerBlocks: Parents and Ancestors”?